src/api/alert.ts

```diff
@@ -0,0 +1,299 @@
+    let callResponse: AppCallResponse;
+
+    try {
+        const message = await ackAlertAction(request.body);
+        callResponse = newOKCallResponseWithMarkdown(message);
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+    }
+
+    response.json(callResponse);
+};
+
+export const unackAlertSubmit = async (request: Request, response: Response) => {
+    let callResponse: AppCallResponse;
+
+    try {
+        const message = await unackAlertCall(request.body);
+        callResponse = newOKCallResponseWithMarkdown(message);
+
+        response.json(callResponse);
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+        response.json(callResponse);
+    }
+};
+
+export const unackAlertModal = async (request: Request, response: Response) => {
+    let callResponse: AppCallResponse;
+
+    try {
+        const message = await unackAlertAction(request.body);
+        callResponse = newOKCallResponseWithMarkdown(message);
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+    }
+
+    response.json(callResponse);
+};
+
+export const otherActionsAlert = async (request: Request, response: Response) => {
+    let callResponse: AppCallResponse = newOKCallResponse();
+
+    try {
+        const other = await otherActionsAlertCall(request.body);
+        if (other !== 'null' && typeof other === 'object') {
+            callResponse = newFormCallResponse(<AppForm>other);
+        }
+        if (typeof other === 'string') {
+            callResponse = newOKCallResponseWithMarkdown(other);
+        }
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+    }
+    response.json(callResponse);
+};
+
+export const closeActionsAlert = async (request: Request, response: Response) => {
+    let callResponse: AppCallResponse;
+
+    try {
+        await closeActionsAlertCall(request.body);
+        callResponse = newOKCallResponse();
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+    }
+
+    response.json(callResponse);
+};
+
+export const addNoteToAlertSubmit = async (request: Request, response: Response) => {
+    let callResponse: AppCallResponse;
+
+    try {
+        const message = await addNoteToAlertCall(request.body);
+        callResponse = newOKCallResponseWithMarkdown(message);
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+    }
+
+    response.json(callResponse);
+};
+
+export const addNoteToAlertModal = async (request: Request, response: Response) => {
+    let callResponse: AppCallResponse;
+
+    try {
+        const message: string = await addNoteToAlertAction(request.body);
+        callResponse = newOKCallResponseWithMarkdown(message);
+    } catch (error: any) {
+        callResponse = showMessageToMattermost(error);
+    }
+
+    response.json(callResponse);
+};
```

Comment:
Why have `*Submit` and `*Modal` if they have the exact same function body? This occurs multiple times in this file